### DISCUSSION TITLE
how to set default active button in 'join' ? just like the button group before.

### DISCUSSION THREAD

**Discussion by businiaoyo** [saadeghi/daisyui#2149] [Q&A]
I want to make Radio1 acitve at first.
<div class="join">
  <input class="join-item btn" type="radio" name="options" aria-label="Radio 1" />
  <input class="join-item btn" type="radio" name="options" aria-label="Radio 2" />
  <input class="join-item btn" type="radio" name="options" aria-label="Radio 3" />
</div>

**Comment by saadeghi** [comment]
Add `checked` attribute to the radio input

Let me know if you have any questions
